gp-cli.md

```diff
@@ -0,0 +1,344 @@
+```
+
+
+#### create-user
+
+Creates a new user in the service instance.
+
+The example below creates a new
+user account with read-only access to the bundles *MyAppsBundle1* and
+*MyAppsBundle2*.
+```
+java -jar gp-cli.jar create-user -t READER -b MyAppsBundle1,MyAppsBundle2 -j mycreds.json
+```
+The output of above command looks like below.
+```
+A new user was successfully created.
+User ID: 9b8b65c31cf56a8eac2bd4a1a1f09175
+User password: IpBiOP0s4xXs3SRSgofYLLasU6/qEWFf
+```
+Note: The password above cannot be retrieved from the service later.
+If you lost the password, you can only reset the password by
+*reset-user-password* command below.
+
+The command option -t specifies a type of new user. There are
+following user types.
+* **ADMINISTRATOR** - Administrator of the service instance.
+* **TRANSLATOR** - Can read bundles and edit tranlations. This account type
+is designed for external tooling for translators.
+* **READER** - Can only read basic information of bundles and resource strings.
+This account type is designed for distributing credentials included in a client
+application accessing the translated strings dynamically.
+
+
+#### reset-user-password
+
+Resets a user's password. The example below resets the password of the
+user whose ID is 9b8b65c31cf56a8eac2bd4a1a1f09175.
+```
+java -jar gp-cli.jar reset-user-password -d 9b8b65c31cf56a8eac2bd4a1a1f09175 -j mycreds.json
+```
+The out of above command looks like below.
+```
+New password: CD0ZPBvOoTCZbDCDssY/sMEjmTvW+rYs
+```
+
+
+#### delete-user
+
+Deletes a user. The example below delets the user whose ID is
+9b8b65c31cf56a8eac2bd4a1a1f09175.
+```
+java -jar gp-cli.jar delete-user -d 9b8b65c31cf56a8eac2bd4a1a1f09175 -j mycreds.json
+```
+
+---
+### <a name="TOC-Cmd-Others"></a>Other Commands
+
+#### list-mt-languages
+
+Prints out machince translation languages currently enabled in the
+service instance. For example, the following example prints out an array
+of machine translation target languages from English (en).
+```
+java -jar gp-cli.jar list-mt-languages -f en -j mycreds.json
+```
+The output of above command looks like below.
+```
+["de","es","fr","it","ja","ko","pt-BR","zh-Hans","zh-Hant"]
+```
+
+---
+## <a name="TOC-How-To"></a>How-To
+
+### <a name="TOC-Import-Existing-Translations"></a>Import Existing Translations
+
+You may already have resource bundles translated from English to several
+different languages. If you want to import the already translated contents to
+a Globalization Pipeline service bundle, then you can follow the steps below.
+
+1\. Create a new bundle with English as the source language. In this example,
+we use *com.ibm.example.MyBundle* as the bundle ID.
+```
+java -jar gp-cli.jar create -b com.ibm.example.MyBundle -l en -j mycreds.json
+```
+
+2\. Import the English(en) resource bundle contents to *MyBundle*. The resource bundle
+file in this example is a Java properties fiel - MyBundle.properties
```

Comment:
`fiel` --> `file` — but probably will submit a follow on PR  for minor copy edit suggestions.